weldx/schemas/weldx.bam.de/weldx/core/transformations/local_coordinate_system-0.1.0.yaml

```diff
@@ -103,20 +103,18 @@ properties:
     tag: "asdf://weldx.bam.de/weldx/tags/time/timedeltaindex-0.1.*"
   coordinates:
     description: |
-      An ndarray containing the coordinates.
+      A ndarray containing the coordinates.
 
     oneOf:
       - tag: "asdf://weldx.bam.de/weldx/tags/core/variable-0.1.*"
       - tag: "asdf://weldx.bam.de/weldx/tags/core/time_series-0.1.*"
   orientations:
     description: |
-      An ndarray containing the orientations.
+      A ndarray containing the orientations.
```

Comment:
Same as above

Reply:
true! I just auto-accepted the PyCharm spelling correction